rockcraft.yaml

```diff
@@ -0,0 +1,46 @@
+name: charmed-mysql # the name of your ROCK
+base: ubuntu:22.04 # the base environment for this ROCK
+version: '0.8.22-1' # just for humans. Semantic versioning is recommended
```

Comment:
The current Dockerfile in branch `8.0-22.04` uses version `0.8.23-1`,
however it is a version of package `mysql-apt-config` while the version of charmed-snap should represent the main payload component version `mysql-server` which has version `8.0.31-0ubuntu2` I would vote to use it here.

What do you think?

Reply:
I agree with this.  To be clear, do we would want to set the version to `8.0.31-0ubuntu2` or `8.0.31` as thats the actual payload version and the rest of it is packaging info.